test/node/conversation-items.js

```diff
@@ -0,0 +1,104 @@
+    });
+
+    it('should add a text item to the conversation and raise an itemAdded event', async () => {
+        const textValue = `${Date.now()}a`;
+        const res  = await Promise.all([
+            client.addTextItem(conversation.convId, textValue),
+            helper.expectEvents(client, [{
+                type: 'itemAdded',
+                predicate: evt => evt.item.convId === conversation.convId && evt.item.itemId
+            }])           
+        ]);
+        item.itemId = res[0] && res[0].itemId;
+        item.content = res[0] && res[0].text.content;
+        assert(res[0] && conversation && res[0].convId === conversation.convId && res[0].text.content === textValue);
+    }); 
+    it('should add a text item to the conversation and raise an itemAdded event', async () => {
+        const textValue = `${Date.now()}b`;
+        const subject = `${Date.now()}c`;
+        const content = {
+            itemId: item.itemId,
+            subject: subject,
+            content: textValue,     
+        }
+        const res  = await Promise.all([
+            client.updateTextItem(content),
+            helper.expectEvents(client, [{
+                type: 'itemUpdated',
+                predicate: evt => evt.item.convId === conversation.convId && evt.item.itemId === item.itemId
+            }])           
+        ]);
+        item.content = res[0] && res[0].text.content;
+        assert(res[0] && res[0].itemId === item.itemId && res[0].text.content === textValue && res[0].text.subject === subject);
+    }); 
+
+    it('should get conversation feed', async () => {
+        const res  = await client.getConversationFeed(conversation.convId);
+        assert(res && res.threads.some(thread => thread.parentItem.convId === conversation.convId && thread.parentItem.itemid === item.itemid));
+    });
+
+    it('should get conversation items', async () => {
+        const res  = await client.getConversationItems(conversation.convId);
+        assert(res.some(conversationItem => conversationItem.itemId === item.itemId));
+    }); 
+ 
+    it('should flag item and get flagged item', async () => {
+        await client.flagItem(conversation.convId, item.itemId);
+        const res = await client.getFlaggedItems();
+        assert(res && res.some(conv => conv.conversationId === conversation.convId && conv.conversationItemData.some(i => i.itemId === item.itemId)));
+    });
+    
+    it('should unflag item', async () => {
+        await client.unflagItem(conversation.convId, item.itemId);
+        const res = await client.getFlaggedItems();
+        assert(res && !res.some(conv => conv.conversationId === conversation.convId && conv.conversationItemData.some(i => i.itemId === item.itemId)));
+    });
+
+    it('should like item', async () => {
+        const res = await client.likeItem(item.itemId);
+        assert(res === undefined || res);
```

Comment:
That is always true

Reply:
How do I test an API that doesn't return anything? Also, I didn't see any API for getFovoriteItems or anything so I can't check that way. Should I just call the API and not assert anythingm or just assert(true) after the API call?